contributing.md

```diff
@@ -0,0 +1,190 @@
+## **Contributing CodeBuddies website version 3 (CBV3)**
+
+## **Setup**
+
+Although it is possible to run the website locally, we recommend you run CodeBuddies locally using [Docker](https://www.docker.com/get-started).  
+
+These instructions have been used on the following operating systems:  
+* Linux
+* Mac OS
+* Windows 10 Pro (Windows 10 Home is supported by Docker Desktop at the moment)
+
+### Prerequisites
+
+  Contributors are expected to have fundamental knowledge of the [technologies used for CBV3](README.md#technologies-used).
```

Comment:
I think we should change the words "fundamental knowledge" to "basic familiarity," so as not to prevent someone who's _interested_ but doesn't fundamentally know the technology from thinking they could get involved.